shared/src/androidMain/kotlin/com/innova/innovaPoolingLibrary/DateFormatter.kt

```diff
@@ -0,0 +1,21 @@
+package com.innova.innovaPoolingLibrary
+
+import android.util.Log
+import java.text.SimpleDateFormat
+import java.util.*
+
+actual class DateFormatter {
+    var selectedDate = Date()
+    actual fun getFormattedDate(): String {
```

Comment:
function name should be named to getCurrentDate because it is formatting the current date